source/process.adoc

```diff
@@ -330,11 +337,13 @@ A Plan Review provides a means for the Specification Team to present their Relea
 [[efsp-reviews-progress]]
 ==== Progress Review
 
-A Specification Project must engage in at least one successful Progress Review during every release cycle. The timing of a Progress Review must coincide with the staging of a Milestone which must be delivered to the PMC and the Specification Committee before the start of the Review.
+Progress Reviews are used to extend a <<efsp-version-lifecycle, Release Cycle>>. During a Release Cycle a Project Team may be required to engage in one or more Progress Reviews. 
+
+The timing of a Progress Review must coincide with the staging of a Milestone which must be delivered to the PMC and the Specification Committee before the start of the Review.
```

Comment:
Again, is this "staging of a Milestone" just talking about the specification document?
Or, like a Release Review, is it talking about all the required artifacts for a release (spec, CI, TCK)?

Reply:
A Milestone Build is described as an "almost release" in the document. My intent is that the Specification Team will build what they have and distribute it via a project channel.

Care must be taken to ensure that Milestone Builds are carefully labeled as such.